Provider.php

```diff
@@ -19,19 +19,25 @@ public function getAccessTokenRefreshResponse ($refreshToken) {
     }
 
     protected function getAuthUrl ($state) {
-        return $this->buildAuthUrlFromBase("https://auth.openticket.tech/token/authorize", $state);
+        $baseUrl = env("OPENTICKETTECH_AUTH_URI", "https://auth.openticket.tech");
```

Comment:
Please use config values instead of env values here

Reply:
Updated